--- a/Common/Core/vtkStructuredPointArray.h
+++ b/Common/Core/vtkStructuredPointArray.h
@@ -74,7 +74,7 @@
   VTK_ABI_NAMESPACE_END                                                                            \
   }
 
-#elif defined(VTK_USE_EXTERN_TEMPLATE)
+#elif defined(VTK_USE_EXTERN_TEMPLATE) && !defined(__MINGW32__)
 #ifndef VTK_STRUCTURED_POINT_ARRAY_TEMPLATE_EXTERN
 #define VTK_STRUCTURED_POINT_ARRAY_TEMPLATE_EXTERN
 #ifdef _MSC_VER
